src/components/ListboxGrid/distribute-resources.ts

```diff
@@ -13,4 +13,48 @@ const distributeResources = (resources: IListboxResource[], columnCount: number)
   return { firstColumn, restOfColumns };
 };
 
-export default distributeResources;
+const removeOverflowingItems = (height: number, foldedItemHeight: number, count: number, rescources: IListboxResource[]) => {
+  const sliceIndex = (height / foldedItemHeight) - count - 2;
+  return rescources.slice(0, sliceIndex);
+};
+
+export const distributeResourcesInFirstCoulumn = (rescources: IListboxResource[], height: number, foldedCount: number, cExpanded: IListboxResource[], cFolded: IListboxResource[]) => {
+  let expanded: IListboxResource[] = [...cExpanded];
+  let folded: IListboxResource[] = [...cFolded];
+  let expandButton = false;
+  const count = rescources.length;
+  const foldedItemHeight = 60; // TODO: Get from FoldedListbox component
```

Comment:
What is blocking from adding this now? Should it come from the container application (sense-client)?

Reply:
Now it's just a constant, but when we add dense mode it will change. I was thinking I could get it from a ref that's pointing to FoldedListbox, then read ref.current.clientHeight from there. But I didn't account for the grid padding, so I will keep it as is for now.